esp/src/src-react/components/FileDetails.tsx

```diff
@@ -0,0 +1,189 @@
+import * as React from "react";
+import { CommandBar, ContextualMenuItemType, ICommandBarItemProps, Pivot, PivotItem, ScrollablePane, ScrollbarVisibility, Sticky, StickyPositionType } from "@fluentui/react";
+import { SizeMe } from "react-sizeme";
+import nlsHPCC from "src/nlsHPCC";
+import * as Utility from "src/Utility";
+import { useFile } from "../hooks/File";
+import { pushUrl } from "../util/history";
+import { ShortVerticalDivider } from "./Common";
+import { TableGroup } from "./forms/Groups";
+
+import "react-reflex/styles.css";
+
+const pivotItemStyle = (size, padding: number = 4) => {
+    if (isNaN(size.width)) {
+        return { position: "absolute", padding: `${padding}px`, overflow: "auto", zIndex: 0 } as React.CSSProperties;
+    }
+    return { position: "absolute", padding: `${padding}px`, overflow: "auto", zIndex: 0, width: size.width - padding * 2, height: size.height - 45 - padding * 2 } as React.CSSProperties;
+};
```

Comment:
Common up with WorkunitDetails - I suspect there is a better way to this as well...